README.md

```
@@ -27,14 +27,30 @@ With a user-friendly command-line interface, this tool is simple yet effective,
    - The `-low`, `-mid`, and `-high` flags are required for specifying the input images.
    - The `-output` flag allows you to specify the name of the output HDR image. If omitted, the default will be `hdr_output.jpg`.
 
-4. **Check Output**:
+4. **Using the TUI**:
+   The application now includes a Text User Interface (TUI) to guide you through the HDR image creation process. To use the TUI, run the following command:
+   ```bash
```

Markdownlint

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)